multi-agent/blueprints/service.py

```
@@ -104,23 +109,26 @@ def list_resolved_docs(
-                # Convert resolved spec to dict
-                resolved_dict = resolved_spec.model_dump(mode="json")
-
-                # Create new doc with resolved spec_dict
-                resolved_doc = dict(doc)  # Copy all fields from original doc
-                resolved_doc["spec_dict"] = resolved_dict  # Replace spec_dict with resolved version
-                resolved_docs.append(resolved_doc)
-            except Exception as e:
+                resolved_docs.append(self._resolve_doc(doc))
+            except Exception:
                 # If resolution fails, skip this document
                 continue
```

Ruff (0.14.14)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)